templates/test.html

```diff
@@ -1,5 +1,26 @@
 {% extends 'base.html' %}
 
 {% block content %}
-    <h1>supports page</h1>
+    <div class="container">
```

Comment:
This template contains Bootstrap classes. It is worth going through bulma.io documentation and using the utility classes provided to replicate what we need, but in Bulma. We aren't using Bootstrap as our framework of choice for this project. 😀